src/docs/cookbook/testing/unit/mocking.md

```diff
@@ -141,7 +141,7 @@ main() {
       when(client.get('https://jsonplaceholder.typicode.com/posts/1'))
           .thenAnswer((_) async => http.Response('{"title": "Test"}', 200));
 
-      expect(await fetchPost(client), isInstanceOf<Post>());
+      expect(await fetchPost(client), TypeMatcher<Post>());
```

Comment:
nit: You can use `const TypeMatcher<Post>()`

Reply:
Added

Reply:
With matcher >= 0.12.5 you can also use [isA](https://pub.dev/documentation/matcher/latest/matcher/isA.html)

```dart
expect(await fetchPost(client), isA<Post>());
```